packages/react-native/ReactCommon/react/renderer/graphics/Transform.cpp

```diff
@@ -240,9 +257,9 @@ Transform Transform::Interpolate(
     result = result *
         Transform::FromTransformOperation(TransformOperation{
             type,
-            lhsOp.x + (rhsOp.x - lhsOp.x) * animationProgress,
-            lhsOp.y + (rhsOp.y - lhsOp.y) * animationProgress,
-            lhsOp.z + (rhsOp.z - lhsOp.z) * animationProgress});
+            ValueUnit(lhsOp.x.value + (rhsOp.x.value - lhsOp.x.value) * animationProgress, UnitType::Point),
```

Comment:
Could you explain why we use `UnitType::Point` unconditionally here?

Reply:
My bad, I assumed it'll be point always. I should be checking whether lhs and rhs have same unit and use that. wdyt of below checks above the result and use one of the op's unit? 
```
react_native_assert(lhsOp.x.unit == rhsOp.x.unit);
react_native_assert(lhsOp.y.unit == rhsOp.y.unit);
react_native_assert(lhsOp.z.unit == rhsOp.z.unit);
```

Reply:
Added the above checks. I think lhs and rhs would have same units during interpolate